<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Data Logger</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignLeft">
    <widget class="QGroupBox" name="serialGroup">
     <property name="title">
      <string>Koneksi Serial</string>
     </property>
     <layout class="QHBoxLayout" name="serialLayout">
      <item>
       <widget class="QLabel" name="label_port">
        <property name="text">
         <string>Port: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="portBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="refresh_ports_btn">
        <property name="toolTip">
         <string>Refresh Port List</string>
        </property>
        <property name="text">
         <string>↻</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_baud">
        <property name="text">
         <string>Baud:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="baudBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="connectBtn">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="loggingGroup">
     <property name="title">
      <string>Pengaturan Logging</string>
     </property>
     <layout class="QHBoxLayout" name="loggingLayout">
      <item>
       <widget class="QLabel" name="label_file">
        <property name="text">
         <string>File:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filepathEdit">
        <property name="placeholderText">
         <string>log.csv</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseBtn">
        <property name="text">
         <string>Browse…</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startBtn">
        <property name="text">
         <string>Start Logging</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="text">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Serial output akan tampil disini ....</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
